File: Java/src/main/java/QuestionBank/Solution101_200/Solution136.java
package QuestionBank.Solution101_200;

/**
 * @author windows
 */
public class Solution136 {
    public int singleNumber(int[] nums) {
        int num = nums[0];
        for (int i = 1; i < nums.length; i++) {
            num = nums[i] ^ num;
        }
        return num;
    }


    public static void main(String[] args) {
        Solution136 solution136 = new Solution136();
    }
}

File: C++/src/Solution101_200/Solution136.cpp
//
// Created by windows on 2023/3/1.
//

#include "Solution136.h"
#include <iostream>
#include <vector>
#include <unordered_set>
#include <unordered_map>
#include <string>
using namespace std;

int Solution136::singleNumber(vector<int> &nums) {
    int num = nums[0];
    for (int i = 1; i < nums.size(); i++){
        num ^= nums[i];
    }
    return num;
}
